### DISCUSSION TITLE
Self Hosting - Auth Config / Settings Unavailable?

### DISCUSSION THREAD

**Discussion by TheoLeeCJ** [supabase/supabase#4444] [Questions] [answered]
After following the self-hosting instructions here (https://supabase.com/docs/guides/hosting/docker) and spinning up a self-hosted Supabase instance, the Auth config options seem to be unavailable in the dashboard:

Hosted Supabase:

![image](https://user-images.githubusercontent.com/22087932/145693610-4e496f67-1ba7-445a-beaa-b85ed77df1b8.png)

Self-Hosted Supabase:

![image](https://user-images.githubusercontent.com/22087932/145693614-7582af87-1068-442a-9a1e-1091753715f2.png)

https://github.com/supabase/supabase/discussions/4421#discussion-3733475 seems to reference a similar issue.

Is this intentional (or maybe a mistake in my setup), or will the config become available at some point? Thank you very much!

The entire Settings tab is missing as well.

![image](https://user-images.githubusercontent.com/22087932/145693694-b32a5b2c-f893-4a9b-a705-5e07b9ea0853.png)

On a side note, the instructions to generate a JWT using the tool here (https://supabase.com/docs/guides/hosting/overview#api-keys) seems to be giving a JWS Invalid Signature issue.

**Comment by gabydd** [comment, marked as answer]
I think that this feature is not available at the moment in the studio, hopefully at some point it will be. I believe there are bunch of features that are turned off if you are on the studio instead of the dashboard(app.supabase.io). Anywhere you see an `IS_PLATFORM` in the code is something they turned off for supabase studio.

**Reply by soedirgo** [reply]
Yes, there are things we're working to make available (see https://github.com/supabase/supabase/issues/4440), but project administration (e.g. Settings page) is out of scope.

Auth config etc. is in the roadmap, but will be configured through a config file rather than from the dashboard.

**Reply by justinr1234** [reply]
@soedirgo is there any update on specifically the auth config?

**Reply by Walledgarden** [reply]
@soedirgo @kiwicopple same question

**Reply by fatherofinvention** [reply]
Is this still the case? If so, what a let down. I had read some pretty critical stuff of Supabase WRT to self-hosting over on HN, and no sooner do I get it spun up on my server do I run into a hard blocker like this. I can only assume it's intentional to drive users to the cloud offering.

**Reply by crimsonhawk47** [reply]
So still nothing?

**Reply by SnapDrive** [reply]
This is very annoying.. In the self hosted version you need to restart the whole docker container only when you want to add new users logins to the supabase studio.

**Reply by miguelgargallo** [reply]
I can access to hidden pages, they just ripped off the functions, maybe if we discover how to deal with .env we will sort this "open source" project to real use https://yourselfhostlink/project/default/auth/providers

**Reply by stobias123** [reply]
Hmm anyone know how to see database settings? auth providers works.

**Reply by miguelgargallo** [reply]
seems they just rippoff to force pay and don't use the free "non opensource" versions

**Reply by MejanH** [reply]
> seems they just rippoff to force pay and don't use the free "non opensource" versions

I was going to recommend supabase to my client since i was calculating estimates for their new project. it looks like next-auth and drizzle orm will do a better job than supabase self hosted app.
